projects/buddy-system.md

```diff
@@ -0,0 +1,98 @@
+
+* Someone from another service team to ensure cross-team integration inside the section
+
+* Can not be the newcomer’s supervisor
+
+* The section leader is responsible of selecting the newcomer's buddy
+
+* Should be available during the first two weeks after the newcomer arrives, and should not have extended absences (i.e. >2-3 weeks) during the first two months
+
+* For absences of a week or more, the buddy should ensure to hand-over buddy responsibilities to a colleague
+
+## What are the responsibilities?
+
+In general buddy's responsibilities are:
+
+1. Being cool with her/his newcomer :sunglasses:
+2. Being available for her/him
+3. Arrange informal meetings with her/him as often as needed
+4. Follow-up with the buddy system maintainer
+5. Turn to the buddy system maintainer any time if needed
+
+Besides, since the process will span over a period of two months, a more specific list divided by phases will be provided.
+
+### Before newcomers arrival
+The buddy must meet the newcomer’s supervisor in order to get an overview of her/his project and discuss the first tasks:
+
+* Schedule when supervisor will introduce buddy to the newcomer
+* Define tasks on Invenio
+  * One of the following small tasks: read up relevant Invenio documentation, rehearse the presentations, test tutorials, check if any part of the material is outdated (~1 hr)
+  * A medium sized task (~1 day) suitable for pair-programming
+  * A first larger service task in which the supervisor will help
+* Agree on who is going to run the bootcamp following `inveniosoftware/training` repository.
+
+### First day
+
+* Take a coffee, present yourself and your project
+* Explain CERN structure: what is a department, section, group etc...
+* Walk through EDH and common actions
+* Let newcomer ask doubts about anything, the more the better
+* Make newcomer aware of:
+	* Invenio Developer Forum
+	* Friday section coffee
+	* Section lunch
+	* Section events (if there are some coming)
+
+
+### 1st week
+
+* Run the bootcamp with the newcomer (using standard material/presentations) (if agreed with supervisor):
+  * Presentation: Introduction to Invenio, use-cases and digital repositories
+  * Presentation: Communication channels (Gitter, Mattermost etc...) Maintainers, Meetings
+* Guided tutorial: Setting up your development environment
+* Guided tutorial: Get Invenio up and running
+* Guided tutorial: Developing for Invenio, including cookiecutter (+seeing all the files), tiny task, running tests, installing inside Invenio
+* Pair-programming: Solve first small task
+* Tutorial: Submitting a pull request
+* Presentation: Overview of contribution and style guide
+* Pair-programming: Solve medium sized task
+* Presentation: Invenio general architecture overview
+* Presentation: Module overview
+* Plan for pair-programming on a service task
+
+**Remember**, newcomers also need “free” time for setting up their environment, office space etc ...
```

Comment:
Shall we say something about being flexible with newcomers also depending on their expertise? There will be some (less experienced) that require more assistant, other ones (more experienced perhaps) that will appreciate the introduction and will prefer to continue on their own till they hit a blocker. WDYT?
It is difficult to express in words, but what I mean is that we should be able to read the newcomer needs, not force her/him to be trained but to understand what level of assistance she/he needs. WDYT @diegodelemos ?